### ISSUE TITLE
Is the project dead? are there any live forks of this great software?

### ISSUE THREAD

**Issue by MichaelHabib** [jichu4n/asciidoclive#13]
Hi,
I see no changes have been made in 2 years & the google drive integration does not work. Is there an up to date fork of this software & what are the plans for this repo?
It would be great if can comment regarding this software or similar ones. thanks.

**Comment by jichu4n** [comment]
No, it's not dead, just haven't had a reason to update.

What do you mean by "google drive integration does not work"?

**Comment by MichaelHabib** [comment]
Hi,
I get "This app isn't verified" , but I did manage to connect to google ... I think having the app verified will create more trust for this website?
But great work non the less, keep it up.

**Comment by jichu4n** [comment]
Ah, thanks for the details! That makes sense. This appears to be related to [a new Google policy](https://gsuite-developers.googleblog.com/2017/07/new-security-protections-to-reduce-risk.html), and I see that Google started enforcing this on asciidoclive.com on 12/04/2017. Will get this fixed.

**Comment by compuguy** [comment]
@jichu4n Any updates on getting Google sign in working again?

**Comment by jichu4n** [comment]
Yes - I'm actively working on a redesign / rewrite of the site and it's close to feature parity. Will have an update for this soon. Appreciate the support @compuguy !

**Comment by silopolis** [comment]
Hello there !

Forgive me for reviving this old one... But I proposed this nice piece of software to be added to an official Awesome AsciiDoc page here https://gitlab.eclipse.org/eclipse-wg/asciidoc-wg/asciidoc.org/-/merge_requests/20 and not sure what I should answer :/

Death or alive ? Plans to upgrade to AsciiDoctor 2 ?

Thanks for the great work and furthermore for sharing it.

TY
J

**Comment by louiss0** [comment]
> Hello there !
> 
> Forgive me for reviving this old one... But I proposed this nice piece of software to be added to an official Awesome AsciiDoc page here https://gitlab.eclipse.org/eclipse-wg/asciidoc-wg/asciidoc.org/-/merge_requests/20 and not sure what I should answer :/
> 
> Death or alive ? Plans to upgrade to AsciiDoctor 2 ?
> 
> Thanks for the great work and furthermore for sharing it.
> 
> TY J

Someone created a [clone](https://asciidocalive.vercel.app/). If you want to render Asciidoc files on a website then use [Astro](https//:astro.build) with [@forastro/asciidoc](https://npmjs.com/package/@forastro/asciidoc).

**Comment by louiss0** [comment]
> Hi, I get "This app isn't verified" , but I did manage to connect to google ... I think having the app verified will create more trust for this website? But great work non the less, keep it up.

Someone created a [clone](https://asciidocalive.vercel.app/). If you want to render Asciidoc files on a website then use [Astro](https//:astro.build) with [@forastro/asciidoc](https://npmjs.com/package/@forastro/asciidoc).
